recipes/mender-client/debian-master/rules

```diff
@@ -7,8 +7,9 @@ override_dh_auto_build:
 	make CGO_CFLAGS="${CGO_CFLAGS}" CGO_LDFLAGS="${CGO_LDFLAGS}" CC="${CC}" \
 	GOOS=linux CGO_ENABLED=1 build
 
-# All install targets except install-systemd
-install_targets = $(shell egrep ^install\-.*: Makefile | grep -v install-systemd | cut -d: -f1)
+install_targets_exclude=install-systemd
+install_targets_all=$(shell grep -E ^install\-.*: Makefile | cut -d: -f1)
```

Comment:
Q: Should not the wildcard be quoted or escaped? :thinking:

Reply:
Pre-existing, but I will fix :+1: